packages/loot-core/src/shared/util.ts

```diff
@@ -231,8 +226,7 @@ export const numberFormats: Array<{
 }> = [
   { value: 'comma-dot', label: '1,000.33', labelNoFraction: '1,000' },
   { value: 'dot-comma', label: '1.000,33', labelNoFraction: '1.000' },
-  { value: 'space-comma', label: '1 000,33', labelNoFraction: '1 000' },
-  { value: 'space-dot', label: '1 000.33', labelNoFraction: '1 000' },
+  { value: 'space-comma', label: '1\xa0000,33', labelNoFraction: '1\xa0000' },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why do we still need to add this invisible char here? Could we just use regular whitespace as before?
```suggestion
  { value: 'space-comma', label: '1 0000,33', labelNoFraction: '1 0000' },
```

Reply:
I updated it here for consistency with the `Intl` API's output. It's not required in this place.